src/components/services/CopyToClipboard.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <Generic :item="item">
+    <template #indicator>
+      <div class="status"><i class="fa-regular fa-copy fa-xl" :class="{'fa-beat': animate, 'copied-icon': animate}" @click="copy"></i></div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "CopyToClipboard",
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  components: {
+    Generic,
+  },
+  data: () => ({
+    animate: false
+  }),
+  methods: {
+    copy() {
+      javascript:navigator.clipboard.writeText(this.item.clipboard)
+      this.animate = true;
+      setTimeout(
+        () => { this.animate = false },
+        1000

```

Comment:
I not against the animation but I think it should be very fast, to it's discreet & subtle, you know what I mean ?

Reply:
As said before, I didn't control animation speed to avoid including a custom CSS animation (which I think it's better, just a fast "blink" or pulse) just because I though you would prefer it that way.

I agree to your comments so I'll include the custom animation in CSS :)